src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -2193,14 +2196,73 @@ void MusicXMLParserPass2::part()
     }
     m_spanners.clear();
 
-    // Clean up unterminated ties
-    for (auto tie : m_ties) {
+    // Clean up any remaining ties
+    for (auto& tie : m_ties) {
         if (tie.second) {
-            cleanupUnterminatedTie(tie.second, m_score, m_pass1.exporterString().contains(u"dolet 6"));
-            m_ties[tie.first] = nullptr;
+            m_unendedTieNotes.push_back(tie.second->startNote());
+            tie.second = nullptr;
```

Comment:
This looks a bit strange to me. a) Don't these ties need to be `delete`d? Or are they deleted elsewhere? Let's just be sure that they do get deleted at some point if they remain unconnected, otherwise by setting nullptr here we loose the pointer, never free the memory location and create a memory leak. b) Wouldn't it make more sense to delete the entry from the map entirely? Does it make sense to have an entry of this map which maps a TieLocation to null?

Reply:
The tie is deleted at the end of `cleanupUnterminatedTie` which is called for ties with no end note when looping over `m_unendedTies`.  
I followed the style the tie commit I ported ages ago used, but I would also prefer deleting the whole entry from the map.  I suppose access _could_ be faster in certain situations (?), but the scale makes this absolutely negligible. I'll change that.